React-Dojo/netflix_clone_react/src/components/MovieTabs.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { classicNameResolver } from 'typescript'
+import classes from './MovieTabs.module.css'
+import { useState } from "react";
+
+type Props = {}
+
+const MovieTabs = (props: Props) => {
+
+    const [activeTab, setActive] = useState<number>(1);
+
+    const setActiveTab = (currentTab: number) => {
+        setActive(currentTab)
+    }
+    return (
+        <div className={classes.movieTabs}>
+            <div className={classes.tabs}>
```

Comment:
If you chose to go with a tabbed interface implementation, a good read and suggestion for markup: https://inclusive-components.design/tabbed-interfaces/#truetabbedinterfaces